### ISSUE TITLE
Question about autogen_config

### ISSUE THREAD

**Issue by parsing-science** [jupyter/notebook#1118]
Hi, I'm at Jupyter Day Chicago. I was trying to use Sphinx to start contributing to the notebook documentation, but the "make html" command failed for me. When I looked in notebook/docs/Makefile, I saw that there was a line: "python3 autogen_config.py", and I don't have Python 3 installed.

Is there a reaon that command needs to be run with Python 3? Could that line be changed to "python autogen_config.py"?

@captainsafia, @willingc

**Comment by parsing-science** [comment]
Yep, it does work locally with Python 2.

Should I update "Contributing to the Documentation" to say that you need Python 3 to contribute to the docs?

**Comment by takluyver** [comment]
Hmm, I'd probably leave it for now. At some point, some of the autogen scripts might actually require Python 3, and then we can definitely say that.

I have both Pythons installed, but Python 3 is the one that's most likely to have up to date packages installed, and the one that runs Sphinx for docs builds, so I prefer these scripts to run with it.

**Comment by takluyver** [comment]
Yeah, I think you can do a note somewhere in the docs.

I wish we had something like the Windows `py` launcher, but ubiquitous on all systems, that would launch your preferred Python version.

**Comment by takluyver** [comment]
Or maybe we could try one and fall back to the other, but it's ugly to do that everywhere:

```
if which python3; then
    python3 ...
else
    python  ...
fi
```

**Comment by willingc** [comment]
@parsing-science Sorry for any confusion. For now, let's put a note in the section about Contributing to Documentation saying that we recommend using Python 3.4+ for building documentation. Though if one is editing documentation Python 2.7.9+ is fine or editing via the GitHub editor. Thanks!

@takluyver Thanks for helping. I've been traveling all day.

**Comment by takluyver** [comment]
Closed by jupyter/jupyter#112
